src/twisted/protocols/test/test_tls.py

```diff
@@ -687,25 +687,6 @@ def connectionMade(self):
 
         return self.writeBeforeHandshakeTest(SimpleSendingProtocol, data)
 
-    def test_writeUnicodeRaisesTypeError(self):
```

Comment:
Maybe leave this test as a reminder that passing bytes should never be supported again.

But I am also ok to have this removed.

Reply:
The problem is it doesn't fail in a useful way now.

Reply:
I think the important part is to make sure that it fails... in any way. Rather than ending up with a case in which the data is ignored... or something like that.

So even if the error is not explicit. I think that it still important to make sure that it fails

Reply:
OK I restored the test.